cdk-infra/lib/constructs/api/webhook-api-construct.ts

```diff
@@ -89,6 +89,7 @@ export class WebhookApiConstruct extends Construct {
       handler: 'HandleJobs',
       environment,
       bundling,
+      timeout,
```

Comment:
Very small, but super important!! The default timeout is only 3 seconds. If the lambda times out before it finishes, SQS will treat that as a failed message read and will attempt to invoke the message again for the lambda to complete properly. If the lambda times out, the SQS queue does not delete the message which means it keeps trying forever.